src/LanguageServer/Protocol/Workspaces/LspWorkspaceManager.cs

```diff
@@ -382,8 +389,23 @@ public void UpdateTrackedDocument(Uri uri, SourceText newSourceText)
             workspaceCurrentSolution = workspace.CurrentSolution;
 
             // Step 3: Check to see if the LSP text matches the workspace text.
+
             var documentsInWorkspace = GetDocumentsForUris(_trackedDocuments.Keys.ToImmutableArray(), workspaceCurrentSolution);
-            if (await DoesAllTextMatchWorkspaceSolutionAsync(documentsInWorkspace, cancellationToken).ConfigureAwait(false))
+            var sourceGeneratedDocuments =
+                _trackedDocuments.Keys.Where(static uri => uri.Scheme == SourceGeneratedDocumentUri.Scheme)
+                    .Select(uri => (identity: SourceGeneratedDocumentUri.DeserializeIdentity(workspaceCurrentSolution, uri), _trackedDocuments[uri].Text))
+                    .Where(tuple => tuple.identity.HasValue)
+                    .SelectAsArray(tuple => (tuple.identity!.Value, DateTime.Now, tuple.Text));
```

Comment:
@CyrusNajmabadi What is the time used for here, generally? Something complicated or in the VS for Windows case just updating the gold bar so the user knows when it was last updated?

Reply:
Think its just the gold bar.  Right now I just picked a time because we need to pass it to `WithFrozenSourceGeneratedDocuments`